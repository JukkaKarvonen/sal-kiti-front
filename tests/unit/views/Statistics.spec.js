import { shallowMount, createLocalVue } from "@vue/test-utils";
import Statistics from "@/views/Statistics.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("Statistics.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(Statistics, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
