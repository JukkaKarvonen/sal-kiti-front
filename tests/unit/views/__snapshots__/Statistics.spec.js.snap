// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statistics.vue renders content 1`] = `
<div>
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <h1>
        statistics.statistics
      </h1>
    </b-col-stub>
  </b-row-stub>
   
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <h2
        class="bg-sal-orange"
      >
        statistics.search
      </h2>
       
      <statisticsform-stub />
    </b-col-stub>
  </b-row-stub>
</div>
`;
